docs/source/dev_guide.rst

```diff
@@ -9,6 +9,20 @@ Developer Guide
 
 .. WARNING:: To create new processes look at examples in Emu_.
 
+
+Re-create a fresh environment
+-----------------------------
+
+.. code-block:: console
+
+  $ make stop  # in case you previously did 'make start'
+  $ conda deactivate  # exit the current 'raven' conda env so we can destroy it
+  $ conda env remove -n raven  # destroy the current conda env to recreate one from scratch
```

Comment:
I think it would be worthwhile to rename the conda env to `ravenwps` (in `environment.yaml`) in order to avoid confusion. Not sure how disruptive this would be in production so that's just a suggestion of course.

Reply:
I've been using `raven-wps` in the rest of the docs. If we change it, we could make the adjustment everywhere (install docs, environment.yml, etc.), so that it's less confusing.

Reply:
Yeah, change should be in a different PR, it involves more than just documentation update.  @cjauvin if you want to open a different issue to we do not forget about it?

Reply:
sure! the repo itself would also benefit from such a change also